pyperf/_system.py

```diff
@@ -950,18 +984,21 @@ def write(self, tune):
 
 
 OPERATIONS = [
+    # Generic operations
     PerfEvent,
     ASLR,
     LinuxScheduler,
     CPUFrequency,
+    IRQAffinity,
+    PowerSupply,
```

Comment:
Add an empty line for readability. Same for the other categories below.

Reply:
I hope I understood what you mean, can you check?

Reply:
Yes, thank you.